lms/templates/support/contact_us.html

```diff
@@ -33,6 +33,9 @@
             'loginQuery': "/login${login_query() | n, js_escaped_string}",
             'dashboardUrl': "${reverse('dashboard') | n, js_escaped_string}",
             'homepageUrl': "${marketing_link('ROOT') | n, js_escaped_string}",
+            'zendeskUrl': "${zendesk_url | n, js_escaped_string}",
```

Comment:
(Pedantic) Naming Nit: This isn't really the `zendeskUrl` - it's really representing a Zendesk API host (like `https://arbisoft.zendesk.com`) right? You could consider using a slightly more descriptive name like `zendeskApiHost`.